src/components/NcActions/NcActions.vue

```diff
@@ -1011,7 +1119,8 @@ export default {
 		return h('div',
 			{
 				class: [
-					'action-item',
+					'action-item action-item--default-popover',
```

Comment:
```suggestion
					'action-item',
```
The class added is nowhere used, no?